packages/main/test/samples/Input.sample.html

```diff
@@ -337,28 +331,25 @@ <h2 class="header-title">Products</h2>
 
 				if (query) {
 					suggestionItems = products
-						.map(function (product) {
-							return product.Name;
-						})
-						.filter(function (productName, index) {
-							return productName.toLowerCase().indexOf(query) === 0;
-						})
+						.filter(({ name }) => name.toLowerCase().indexOf(query) === 0)
```

Comment:
You can keep the ES6 syntax in the second part meant for the users, but here the actual script that is executed should be ES5, otherwise the sample won't run on IE11, as we currently don't transpile the playground resources